arm9/lib/src/RTC_internal.c

```diff
@@ -0,0 +1,32 @@
+#include "global.h"
+#include "PXI_fifo.h"
+#include "RTC_internal.h"
+
+u32 RTCi_ReadRawDateTimeAsync(void)
+{
+    return RtcSendPxiCommand(16);
+}
+
+u32 RTCi_ReadRawDateAsync(void)
+{
+    return RtcSendPxiCommand(17);
+}
+
+u32 RTCi_ReadRawTimeAsync(void)
+{
+    return RtcSendPxiCommand(18);
+}
+
+u32 RTCi_WriteRawStatus2Async(void)
+{
+    return RtcSendPxiCommand(39);
+}
+
+u32 RtcSendPxiCommand(u8 command)
```

Comment:
This function is probably a `BOOL`

Reply:
Yup, matched as BOOL. Applied it to rest of `RTC_internal`. Thanks!